apps/condo/pages/auth/signin.tsx

```diff
@@ -1,170 +1,172 @@
+    mutation authenticateUserWithPhoneAndPassword ($phone: String!, $password: String!) {
+        obj: authenticateUserWithPhoneAndPassword(data: { phone: $phone, password: $password }) {
             item {
                 id
             }
         }
     }
-`
+` 
 
-const SignInForm = ({ firebaseUser, children, ExtraErrorToFormFieldMsgMapping = {} }) => {
-    const [form] = Form.useForm()
+const INPUT_STYLE = { minWidth: '273px' }
+const LINK_STYLE = { color: '#389E0D' }
+const SMALL_TEXT = { fontSize: '12px', lineHeight: '20px', color: 'gray' }
+
+const LoginPage = (): React.ReactElement => {
     const intl = useIntl()
-    const [isLoading, setIsLoading] = useState(false)
-    const { refetch } = useAuth()
-    const [signin] = useMutation(SIGNIN_BY_FIREBASE_ID_TOKEN_MUTATION)
-    let initialValues = getQueryParams()
-    initialValues = { ...initialValues, password: '', confirm: '', captcha: 'no' }
+    const SignInTitleMsg = intl.formatMessage({ id: 'pages.auth.SignInTitle' })
+    return (
+        <>
+            <Head>
+                <title>{SignInTitleMsg}</title>
+            </Head>
+            <div>
+                <Typography.Title>{SignInTitleMsg}</Typography.Title>
+                <LoginForm />
+            </div>
+        </>
+    )
+}
 
+
+const LoginForm = (): React.ReactElement => {
+    const [form] = Form.useForm()
+    const { next } = getQueryParams()
+    const { refetch } = useAuth()
+    const initialValues = { password: '', phone: '' }
+    const intl = useIntl()
     const SignInMsg = intl.formatMessage({ id: 'SignIn' })
-    const RegisterMsg = intl.formatMessage({ id: 'Register' })
-    const LoggedInMsg = intl.formatMessage({ id: 'pages.auth.LoggedIn' })
-    const EmailIsNoFoundMsg = intl.formatMessage({ id: 'pages.auth.EmailIsNoFound' })
-    const WrongPasswordMsg = intl.formatMessage({ id: 'pages.auth.WrongPassword' })
-    const UserIsNotFoundMsg = intl.formatMessage({ id: 'pages.auth.UserIsNotFound' })
+    const ExamplePhoneMsg = intl.formatMessage({ id: 'example.Phone' })
+    const PasswordMsg = intl.formatMessage({ id: 'pages.auth.signin.field.Password' })
+    const PhoneMsg = intl.formatMessage({ id: 'pages.auth.register.field.Phone' })
+    const ResetMsg = intl.formatMessage({ id: 'pages.auth.signin.ResetPasswordLinkTitle' })
+    const UserNotFound = intl.formatMessage({ id: 'pages.auth.UserIsNotFound' })
+    const PasswordMismatch = intl.formatMessage({ id: 'pages.auth.WrongPassword' }) 
+
+    const [isLoading, setIsLoading] = useState(false)
+    const [signinByPhoneAndPassword] = useMutation(SIGNIN_BY_PHONE_AND_PASSWORD_MUTATION)
     const ErrorToFormFieldMsgMapping = {
-        [WRONG_EMAIL_ERROR]: {
-            name: 'email',
-            errors: [EmailIsNoFoundMsg],
+        '[notfound.error]': {
+            name: 'phone',
+            errors: [UserNotFound],
         },
-        [WRONG_PASSWORD_ERROR]: {
+        '[passwordAuth:secret:mismatch]': {
```

Comment:
extract the error const inside user/constants

Reply:
Skipped. Need to talk about error constants